openedx/features/learner_profile/tests/views/test_learner_profile.py

```diff
@@ -20,6 +21,7 @@
 
 
 @ddt.ddt
+@override_switch('student-records', True)
```

Comment:
nit: I looked around and saw other switches all using 🐍 case

```
# Switches
INVALIDATE_CACHE_ON_PUBLISH = u'invalidate_cache_on_publish'
STORAGE_BACKING_FOR_CACHE = u'storage_backing_for_cache'
RAISE_ERROR_WHEN_NOT_FOUND = u'raise_error_when_not_found'
```

I'm sorry :(

Reply:
Guh, you're right. Will change.